platforms/android/library-compose/src/main/java/io/element/android/wysiwyg/compose/RichTextEditor.kt

```diff
@@ -23,9 +27,32 @@ fun RichTextEditor(
     modifier: Modifier = Modifier,
 ) {
     val context = LocalContext.current
+    val lifecycleOwner = LocalLifecycleOwner.current
+
+    // Clean up the connection between view and state holder
+    DisposableEffect(lifecycleOwner) {
```

Comment:
This can be done without lifecycle.
```
DisposableEffect(Unit) {
onDispose {
state.viewConnection = null
}
}
```
This will set `viewConnection` as null whenever `RichTextEditor` leaves the composition.